server/plugins/prismaPlugin.js

```diff
@@ -2,7 +2,9 @@ import fp from 'fastify-plugin';
 import { PrismaClient } from '@prisma/client';
 
 const prismaPlugin = fp(async (server, _options) => {
-  const prisma = new PrismaClient();
+  const prisma = new PrismaClient({
+    datasourceUrl: process.env.DATABASE_URL,
```

Comment:
By passing in the DATABASE_URL explicitly in the creation of the PrismaClient, we can override that url between the development server and tests.